centreon/doc/API/centreon-api-v23.10.yaml

```diff
@@ -657,6 +657,31 @@ paths:
         '500':
```

Comment:
Missing release note on top of the file

Reply:
Omg, you're right.
I always forget this 🤦‍♂️️